bip-XXXX.mediawiki

```diff
@@ -768,6 +773,8 @@ TODO: Something about how having only one pool server is great cause you can mul
 ==Discussion==
 
 ; Why TCP? Isn't UDP better for low-latency relay?
+; please take a look for UDP+ICE/STUN & IGMP as well.
```

Comment:
Added a further discussion at https://github.com/TheBlueMatt/bips/blob/betterhash/bip-XXXX.mediawiki#Discussion is that sufficient?